src/WorkflowCore/Interface/IWorkflowMiddlewareErrorHandler.cs

```diff
@@ -11,8 +12,9 @@ public interface IWorkflowMiddlewareErrorHandler
         /// <summary>
         /// Asynchronously handle the given exception.
         /// </summary>
+        /// <param name="workflowInstance">Workflow instance where error happened</param>
         /// <param name="ex">The exception to handle</param>
         /// <returns>A task that completes when handling is done.</returns>
-        Task HandleAsync(Exception ex);
+        Task HandleAsync(WorkflowInstance workflowInstance, Exception ex);
```

Comment:
How can we make this backward compatible?

Reply:
@danielgerlag I can introduce something like `IWorkflowMiddlewareErrorHandlerV2` that will have an old and a new method. And update the usages. However, I don't like this approach. What do you think?